jdaviz/core/registries.py

```diff
@@ -109,16 +130,41 @@ def add(self, name, cls, label=None, icon=None):
             The label displayed in the tooltip when hovering over the tray tab.
         icon : str, optional
             The name of the icon to render in the tray tab.
+        viewer_requirements : str, list of str
+            Required viewers for this plugin.
         """
         if name in self.members:
             raise ValueError(f"Viewer with the name {name} already exists, "
                              f"please choose a different name.")
         else:
             # store the registry name/label so we can access them from the instantiated
             # objects (when determining if a specific plugin is open, for example)
+            viewer_reference_name_kwargs = {}
+
+            if viewer_requirements is not None:
```

Comment:
could this whole if statement be avoided if `viewer_requirements` defaults to an empty list instead of `None`?